src/frontend/mame/clifront.cpp

```diff
@@ -568,27 +568,27 @@ void cli_frontend::listroms(const char *gamename)
 			osd_printf_info("\n");
 		first = false;
 		osd_printf_info("ROMs required for driver \"%s\".\n"
-				"Name                    Size Checksum\n", drivlist.driver().name);
+				"Name                                   Size Checksum\n", drivlist.driver().name);
 
 		// iterate through roms
 		for (device_t &device : device_iterator(drivlist.config().root_device()))
 			for (const rom_entry *region = rom_first_region(device); region; region = rom_next_region(region))
 				for (const rom_entry *rom = rom_first_file(region); rom; rom = rom_next_file(rom))
 				{
 					// accumulate the total length of all chunks
-					int length = -1;
+					uint32_t length = 0;
```

Comment:
Is this actually safe?  Are there situations where 0 was a valid output?  If so, you need to use the next signed type up, i.e. make it `uint64_t`, so it can hold any `uint32_t` and also the negative sentinel value.